### ISSUE TITLE
Holograms turned into ton of armor stands randomly

### ISSUE THREAD

**Issue by GavinGoGaming** [DecentSoftware-eu/DecentHolograms#202]
### Just making sure

- [X] I made sure, this bug is not already known or reported.
- [X] I believe, that this bug is fixable.

### Reproduction

Unsure how it happened, i just logged in one day and it was like this.
![image](https://github.com/DecentSoftware-eu/DecentHolograms/assets/57292172/a0b30387-ceca-422d-829b-4715bd2bed7a)

### Solution

Some issue loading hologram maybe?

### Server Version

1.20.4

### Client Version

1.20.4

### Plugin Version

2.8.6

### Log

https://mclo.gs/kqJbgPx

**Comment by Andre601** [comment]
Outside of some outdated plugins (You should update. Also, don't use Plugman... Reloading plugin is never a good idea, no matter the reasons) is ther enothing unusual in the logs, so my best guess is, that this is simply a client-side issue.

**Comment by GavinGoGaming** [comment]
I use plugman specifically for developing plugins - not for plugins downloaded from others. I dont think this is client-side as it occurs to me and my friends. Reloading dh via /dh reload doesnt help too. Even doing `/minecraft:kill @e[type=minecraft:armor_stand]` does not kill these armor stands, meaning they are being generated via dh.

**Comment by Andre601** [comment]
Yes, the armor stands are made by DH, hence why the kill command doesn't work.

My best guess right now is, that it is an issue with lag. Your logs show that your server has trouble keeping up at times. And perhaps it's that causing problems, where the server skips sending specific packets, like those telling the client to hide the armor stands...

Only other cause could be an issue with another plugin, but that is less likely from what I know.

**Comment by GavinGoGaming** [comment]
Heres a video of it happening: https://youtu.be/z5ssFkXyIXg
This video was made when the server is at 20 TPS with 1 player online.

Looking at the logs, the server overloaded messages only appear when first startup occurs, aka when ImageFrame is loading the map URLs (makes sense for loading like 50 images at once). Only other lag messages i can find are moved wrongly/moved too fast which comes from a clientside flight boost mod (built into many clients, including lunar)

**Comment by techboy04** [comment]
Do you so happen to have ViaRewind? Ive installed it which gave me the same issue you had and after deleting it, it fixed it.

**Comment by GavinGoGaming** [comment]
Yes - I do

**Comment by techboy04** [comment]
Try removing it, and see if theyre still there.

**Comment by GavinGoGaming** [comment]
I removed viarewind and still no fix. Adding it back does nothing either.
![image](https://github.com/DecentSoftware-eu/DecentHolograms/assets/57292172/09c07f68-ddc9-4f95-b908-813740a35b95)

**Comment by Andre601** [comment]
What you should try:

- Remove ViaVersion (not just ViaBackwards)
- Stop and start the server when doing that. Don't use PlugMan for that, as it breaks more than it fixes (In general, don't use PlugMan)

**Comment by evyatar1298** [comment]
This is because of the protocolib

**Comment by evyatar1298** [comment]
try to remove him

**Comment by GavinGoGaming** [comment]
> What you should try:
> 
> - Remove ViaVersion (not just ViaRewind)
> - Stop and start the server when doing that. Don't use PlugMan for that, as it breaks more than it fixes (In general, don't use PlugMan)

I'll try this in a bit. As I said before, plugman is only used for hot-reloading **my personal developed plugins** without full rebooting.

**Comment by KenterWasTaken** [comment]
I have the same issue, but also that some holograms switch places with others, I also use DH

**Comment by d0by1** [comment]
More than one person mentioned it was caused by ProtocolLib so I suppose this is resolved? To streamline the issue tracker, I will now close this. If you experience this issue again, please create a new issue. Thanks.
